Scripts/Utilities/InputUtils.cs

```diff
@@ -42,6 +42,31 @@ public static void CollectDeviceSlotsFromActionMapInput(ActionMapInput actionMap
 				deviceSlots.Add(deviceSlot);
 			}
 		}
+
+		public static void GetBindingDictionaryFromActionMap(ActionMap actionMap, Dictionary<string, List<VRInputDevice.VRControl>> bindingDictionary)
```

Comment:
Nice

Reply:
Heh, yeah it's ugly but it works. I still don't like that I need to hardcode action names, but I couldn't think of a better system. Also, I bet this (and the code for locking/consuming controls) has to change when we update to the new system.